src/App.jsx

```diff
@@ -1,107 +1,30 @@
 import './App.scss';
-
-// import postsFromServer from './api/posts.json';
-// import commentsFromServer from './api/comments.json';
-// import usersFromServer from './api/users.json';
+import { PostList } from './components/PostList';
+import postsFromServer from './api/posts.json';
+import commentsFromServer from './api/comments.json';
+import usersFromServer from './api/users.json';
+
+function getUserById(userId) {
+  return usersFromServer
+    .find(user => user.id === userId)
+    || null;
```

Comment:
You don't need to manually return `null`, if `find` doesn't find anything, it will return `undefined`.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find